docs/reference/platforms/linux/system.rst

```diff
@@ -0,0 +1,243 @@
+will be the version of Python used by the bundled app. However, when building a
+system package, Briefcase will use the operating system's Python3 installation
+for system packages, regardless of the host Python version. This means you
+will need to perform additional platform testing to ensure that your app is
+compatible with that version of Python.
+
+Icon format
+===========
+
+Deb packages uses ``.png`` format icons. An application must provide icons in
+the following sizes:
+
+* 16px
+* 32px
+* 64px
+* 128px
+* 256px
+* 512px
+
+Splash Image format
+===================
+
+Linux System packages do not support splash screens or installer images.
+
+Additional files
+================
+
+The Linux system app template includes a ``LICENSE`` and ``CHANGELOG`` file,
+with stub content. When the application is generated from template, Briefcase
+will look in the project root folder (i.e., the folder that contains your
+``pyproject.toml``) for files with the same name. If these files are found, they
+will be copied into your project. You should ensure these files are complete and
+correct before publishing your app.
+
+The Linux system app template also includes an initial draft manfile for your
+app. This manfile will be populated with the ``description`` and
+``long_description`` of your app. You may wish to add more details on app usage.
+
+Additional options
+==================
+
+The following options can be provided at the command line when producing
+Deb packages.
+
+``--target``
+~~~~~~~~~~~~
+
+A Docker base image identifier for the Linux distribution you want to target.
+The identifier will be in the pattern ``<vendor>:<codename>`` (e.g.,
+``debian:buster`` or ``ubuntu:jammy``). You can also use the version number in
+place of the codename (e.g., ``debian:10``, ``ubuntu:22.04``, or ``fedora:37``).
+Whichever form you choose, you should be consistent; no normalization of
+codename and version is performed, so ``ubuntu:jammy`` and ``ubuntu:22.04`` will
+be identified as different versions (even though they the same version).
+
+You can specify any identifier you want, provided the distribution is still
+supported by the vendor, and system Python is Python 3.8 or later.
+
+The following Linux vendors are known to work as Docker targets:
+
+  * Debian (e.g., ``debian:bullseye`` or ``debian:11``)
+  * Ubuntu (e.g., ``ubuntu:jammy`` or ``ubuntu:22.04``)
+  * Fedora (e.g, ``fedora:37``)
+  * AlmaLinux (e.g., ``almalinux:9``)
+  * Red Hat Enterprise Linux (e.g., ``redhat/ubi9:9``)
+  * Arch Linux (e.g., ``archlinux:latest``)
+  * Manjaro Linux (e.g., ``manjarolinux/base:latest``)
+
+Application configuration
+=========================
+
+To generate a Linux system package, an application *must* define a
+``long_description``. This ``long_description`` *must not* be a copy of the
+short ``description``; nor can the first line of the ``long_description`` match
+the ``description``.
```

Comment:
I don't think there's any need to repeat this here: if the user has generated their project from the current briefcase-template, then they'll already have a `long_description`, and if they haven't regenerated it, then they won't be able to build a package anyway, as I discovered yesterday.